child_update_picture_date/wizard/__init__.py

```diff
@@ -0,0 +1,12 @@
+# -*- encoding: utf-8 -*-

```

Comment:
A new module only to run a cron job? I think this would be acceptable to include this in child_compassion.